package.json

```diff
@@ -1,6 +1,9 @@
 {
-    "name": "fastify-autoroutes",
-    "version": "3.0.2",
+    "name": "@jvddavid/fastify-autoroutes",
```

Comment:
can you revert to previous package name?

Reply:
yes, i changed it just to use at the moment